lib/crypto/crc32.toit

```diff
@@ -14,26 +14,17 @@ The $data must be a string or byte array.
 Returns the checksum as a 4 element byte array in little-endian order.
 */
 crc32 data from/int=0 to/int=data.size -> ByteArray:
```

Comment:
I would make the initial-state and xor_result be optional parameters. (unless nobody uses any else).

For crc32, the function might be good enough, as there aren't many different implementations.